File: 137. Single Number II/solution.c
int singleNumber(int* nums, int numsSize){
     int count=0;
        for(int i=0;i<numsSize;i++){
            for(int j=0;j<numsSize;j++){
                if(i!=j){
                     if(nums[i]==nums[j]){
                        count++;
                        }
                    }
                }
             if(count==0){
                 return nums[i];
             }
             count=0;
        }
        return 1;

}

File: 137. Single Number II/solution.java
class Solution {
    public int singleNumber(int[] nums) {
        int count=0;
        Arrays.sort(nums);
        for(int i=1;i<nums.length;i++){
            if(nums[i]==nums[i-1]){
                count++;
            }
            else{
                if(count<2){
                    return nums[i-1];
                }
                count=0;
            }
        }
        return nums[nums.length-1];
    }
}


